MADE.App.Design/MADE.App.Design.csproj

```diff
@@ -52,12 +52,11 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="docfx.console" Version="2.36.0" PrivateAssets="All" />
```

Comment:
What are the reasons for removing docfx?

Reply:
This was because of issues with it in the multi-targeting world. Sometimes when building the project, the log file would be locked and cause MSBuild to fail to build.